Python/tigre/utilities/im3Dnorm.py

```diff
@@ -34,6 +34,7 @@ def im3DNORM(img,normind,varargin=None):
     if type(normind) is int:
         return np.linalg.norm(img.ravel(),normind)
     if normind == 'TV':
-        raise Exception('TV not yet implemented')
-
+        gx,gy,gz = np.gradient(img.ravel())

```

Comment:
This doesnt work for 2 reasons. 

1. if you `ravel` the array, then it doesn't have `gy` and `gz`, its 1D.
2. `np.gradient` uses central differences. A checkerboard pattern will have zero TV in this case. We need "the same" TV as its used in the source, i.e. backwards differences.

Reply:
I see. Then maybe we should go back to np.diff, following MATLAB.